agent/hcp/deps.go

```diff
@@ -25,18 +24,28 @@ type Deps struct {
 	Sink     metrics.MetricSink
 }
 
-func NewDeps(cfg config.CloudConfig, logger hclog.Logger, nodeID types.NodeID) (Deps, error) {
+func NewDeps(cfg config.CloudConfig, logger hclog.Logger) (Deps, error) {
+	logger = logger.Named("sink")
```

Comment:
nit: moving this up here would make the Scada init logs have a named prefix of `sink.scada`. We could move it to line 41